src/apps/Permissions/Permissions.js

```diff
@@ -169,12 +177,30 @@ class Permissions extends React.Component {
             )}
           </Screen>
 
-          <Screen position={1}>
-            {['entity', 'app'].includes(location.screen) && (
-              <PermissionsList
-                appsLoading={appsLoading}
-                permissions={permissionsDemo.appPermissions}
-                onEdit={this.editPermission}
+          <Screen position={1} animate={animateScreens}>
+            {location.screen === 'app' && (
+              <AppPermissions
+                loading={appsLoading}
+                app={location.app}
+                permissions={permissions}
+                onRevoke={this.revokePermission}
+                daoAddress={daoAddress}
+                resolveRole={resolveRole}
+                resolveEntity={resolveEntity}
+              />
+            )}
+          </Screen>
+
+          <Screen position={1} animate={animateScreens}>
```

Comment:
I'd say now that there's two `Screen`s with `position={1}`, it looks a bit odd. What do you think about using a `switch (location.screen)` to collapse these two?

Reply:
I’m not sure I get this? They are both in position 1 (relatively to the home screen).

Reply:
I think it would've been natural to increment the positions with every new `Screen`

Reply:
I see the hierarchy like this:

Home (position 0)
  - App permissions (position 1)
  - Entity permissions (position 1)

It’s also how the header navigation is defined, with an array of two items max:

- Home: `['Permissions']`
- App permissions: `['Permissions', 'App name / identifier']`
- Entity permissions: `['Permissions', 'Entity permissions / entity']`

Do you think it should be done differently?

Reply:
The hierarchy makes sense, but reusing the same `Screen` element would be clearer that they're grouped like that.

Reply:
Oh yes I can do that to remove an instance of `Screen`. The idea was to separate every “screen” in a separate `<Screen />`… but we are only talking about 3 screens here, so I guess it’s ok to do it this way.